department-of-veterans-affairs/va.gov-cms
# VACMS-3465: Add spotlight panel.

## Description
See #3465 

## Testing done
Visual / Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/100017279-a7542f80-2da8-11eb-8528-5183a9980fe2.png)
![image](https://user-images.githubusercontent.com/2404547/100017291-aae7b680-2da8-11eb-9092-fb708ab9abc3.png)

## QA steps
- As an administrator, go to `/node/add/campaign_landing_page` and click on the Spotlight panel
- Visually verify that only a `Spotlight opt-in` toggle appears
- Visually verify that clicking it reveals all section fields, and text fields are required
- Add list of link teasers, and visually verify that after third is added, button to add more no longer appears

### Definition of Done


### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
